lib/routes/jlu/phy/xzgz/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Route } from '@/types';
+import { handler } from '../handler';
+
+export const route: Route = {
+    path: '/phy/xzgz/:category',
```

Comment:
Parameter description should be [an object](https://docs.rsshub.app/joinus/new-rss/start-code#creating-route)